### PR TITLE
Changed source of package to patch project

### PR DESCRIPTION
**Pull request by Dinno**

### CODE REVIEW

File: cmake/projects/freetype/hunter.cmake [hunk 1]
```diff
@@ -15,9 +15,9 @@ hunter_add_version(
     VERSION
     "2.6.2"
     URL
-    "http://sourceforge.net/projects/freetype/files/freetype2/2.6.2/freetype-2.6.2.tar.bz2/download"
+    "https://github.com/hunter-packages/freetype/archive/hunter.tar.gz"
```
Comment (ruslo) [inline cmake/projects/freetype/hunter.cmake]: This file will change from time to time. For example if I add some commit to branch `hunter` then SHA1 of archive will change => download will fail with hash mismatch. You need to create release and add stable/release archive here.

Comment (Dinno) [inline cmake/projects/freetype/hunter.cmake]: I will do it. But i want to fix the problem with test build first.

**Comment by Dinno** [comment]
A fixed build and changed source of package to tag v2.6.2-p0

**Comment by ruslo** [comment]
Merged. Please set destination branch `develop` next time, see https://github.com/ruslo/hunter/wiki/dev.contribution. Thanks!

**Comment by ruslo** [comment]
https://github.com/ruslo/hunter/releases/tag/v0.12.34

**Pull request merged**